cmd/admin-heal-ui.go

```diff
@@ -393,46 +396,70 @@ func (ui *uiData) UpdateDisplay(s *madmin.HealTaskStatus) (err error) {
 	return
 }
 
-func (ui *uiData) DisplayAndFollowHealStatus() (err error) {
+func (ui *uiData) healResumeMsg(aliasedURL string) string {
+	var flags string
+	if ui.HealOpts.Recursive {
+		flags += "--recursive "
+	}
+	if ui.HealOpts.DryRun {
+		flags += "--dry-run "
+	}
+	if ui.ClientToken != "" {
+		flags += fmt.Sprintf("--client-token %s ", ui.ClientToken)
+	}
+	if ui.Bucket != "" {
+		aliasedURL += "/" + ui.Bucket
+	}
+	if ui.Prefix != "" {
+		aliasedURL += "/" + ui.Prefix
+	}
+	return fmt.Sprintf("Healing is backgrounded, to resume watching use `mc admin heal %s %s`", flags, aliasedURL)
+}
+
+func (ui *uiData) DisplayAndFollowHealStatus(aliasedURL string) (err error) {
 	var res madmin.HealTaskStatus
+	trapCh := signalTrap(os.Interrupt, syscall.SIGTERM, syscall.SIGKILL)
+	trapMsg := ui.healResumeMsg(aliasedURL)
 
 	firstIter := true
 	for {
-		_, res, err = ui.Client.Heal(ui.Bucket, ui.Prefix, *ui.HealOpts,
-			ui.ClientToken, ui.ForceStart)
-		if err != nil {
-			return err
-		}
+		select {
+		case <-trapCh:
+			return errors.New(trapMsg)
```

Comment:
For better UX, I think you need to return errUserInterrupted here and construct the command in the caller so it can show a better error message:

This is the current situation:

```
mc: <ERROR> Unable to display follow heal status. Healing is backgrounded, to resume watching use `mc admin heal --recursive --client-token 0d966db8-78c1-4eb3-9dab-abf1ed79ed19  myminio/testbucket//testbucket`
```

Reply:
Client-token is not needed anymore.